force-app/main/default/classes/Homework.cls

```diff
@@ -63,38 +82,50 @@ public with sharing class Homework {
     {
         // Declare two Integers, with the values 6 and 7, in a single 
         // statement.
+        Integer firstValue = 6, secondValue = 7;
     }
 
     public void booleanFromString()
     {
         // Create a boolean whose value is initialised from the text
         // ‘true’.
+        
+        Boolean myBooleanValue = Boolean.valueOf('true');
+
     }
 
     public void reverse()
     {
         // Create a string with the value ‘1234567890’ 
-        
+        String leftToRight = '1234567890';
         // Use a built-in method to convert to ‘0987654321’
+        String rightToLeft = leftToRight.reverse();
     }
 
     public void findApex(String candidate) 
     {
         // Find the position of the substring “Apex” in candidate
+        //Bhargava is learning Apex programming
+        String apex = 'Apex';
+        Integer substringPosition = 'Bhargava is learning Apex programming'.indexOf(apex);
+
     }
 
     public void get10And15(String candidate)
     {
         // Extract the 10th and 15th characters from a string
-
+        String newString = 'MoreThan15Characters';
+        String newSubStringedString = newString.substring(10, 15);
```

Comment:
This will return a 5 character string, not the 10th and 15th characters. As the comment asked for two things, you will need to call substring twice - once to get the 10th character and once to get the 15th.

Also, note that the first character of a string starts at position 0, not 1.